src/contexts/UrlContextProvider.tsx

```diff
@@ -212,6 +213,14 @@ const getWindowUrlHashParams = () => {
             parsed;
     }
 
+    const isPrettified = hashParams.get(HASH_PARAM_NAMES.IS_PRETTIFIED);
+    if (null !== isPrettified) {
+        urlHashParams[HASH_PARAM_NAMES.IS_PRETTIFIED] = "true" === isPrettified;
+    } else {
+        // Default is disable prettifying
+        urlHashParams[HASH_PARAM_NAMES.IS_PRETTIFIED] = false;
+    }
```

Comment:
if you agree with the above proposal of changing the default:
```suggestion
    const isPrettified = hashParams.get(HASH_PARAM_NAMES.IS_PRETTIFIED);
    if (null !== isPrettified) {
        urlHashParams[HASH_PARAM_NAMES.IS_PRETTIFIED] = "true" === isPrettified;
    }
```

Reply:
Fixed!